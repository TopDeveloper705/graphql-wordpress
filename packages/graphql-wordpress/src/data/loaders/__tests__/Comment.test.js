import getCommentLoaders from 'data/loaders/Comment';

const Comment = getCommentLoaders();

describe('Test Comment loader', () => {
  test('Load a comment', async () => {
    const comment = await Comment.load(13);
    expect(comment.getID()).toMatchSnapshot();
    expect(comment).toMatchSnapshot();
  });
});
