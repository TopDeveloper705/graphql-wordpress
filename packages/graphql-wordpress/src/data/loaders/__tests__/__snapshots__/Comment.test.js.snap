// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Comment loader Load a comment 1`] = `"Q29tbWVudDoxMw=="`;

exports[`Test Comment loader Load a comment 2`] = `
Comment {
  "id": 13,
}
`;
